.github/CODEOWNERS

```diff
@@ -3968,6 +3968,7 @@
 /types/set-value/                               @DanielRosenwasser
 /types/settings/                                @shreyjain1994
 /types/sha1/                                    @arcdev1
+/types/shallow-equals/                          @rsolomon
```

Comment:
The codeowners file will be updated automatically on publish, so please remove this edit.